Testscripts/Windows/VALIDATE-DA.ps1

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+<#
+.Synopsis
+    This script validates Dependency Agent.
+
+.Description
+    The script will install the Latest Dependency Agent and validate install/uninstall
+    scenarios as well as enable/disable scenarios.
+'
+#>
+param([String] $TestParams,
+      [object] $AllVmData)
+function Main {
+    param (
+        [String] $VMName,
+        [String] $Ipv4,
+        [int] $VMPort,
+        [String] $VMUserName,
+        [String] $VMPassword
+    )
+
+    $remoteScriptName = "validate-da"
+    $remoteScriptType = "sh"
+    #######################################################################
+    #
+    #	Main body script
+    #
+    #######################################################################
+
+    # Checking the input arguments
+    if ($VMName.length -eq 0) {
+        Write-LogErr "VM name is empty!"
+        return "FAIL"
+    }
+
+    # Running validate-da script
```

Comment:
Like other PS1 script, can you add 'try-catch' block.

Reply:
Done